crates/wasmtime-backend/src/caller.rs

```diff
@@ -25,6 +25,15 @@ use marine_wasm_backend_traits::prelude::*;
 use wasmtime::AsContext as WasmtimeAsContext;
 use wasmtime::AsContextMut as WasmtimeAsContextMut;
 
+// these imports are used in the macro `impl_func_getter`, but compiler does not detect it
+#[allow(unused)]
```

Comment:
I think it'd be better to move them inside the `get_func` function in this macro.